NuCivic/recline.view.nvd3.js
# Add option to change chart height CIVIC-6001

issue: CIVIC-6001

## Description
Creating a chart with a large number of items on the y axis gets squished looking as the chart has a set height of 480px. To allow the user to adjust the height we need a form field on the chart config to capture this setting.

## Steps to reproduce
1. Create a horizontal bar chart with the attached file.
2. Confirm that the left side labels are compacted and hard to read.

## QA Steps
1. Create a horizontal bar chart with the attached file.
2. Confirm that the left side labels are compacted and hard to read.
3. Edit the chart and enter a height value such as 850
4. Confirm that the labels now have some breathing room
5. View the chart with IE, and confirm the chart also displays in that browser

## Related PRs
https://github.com/NuCivic/visualization_entity/pull/61
https://github.com/NuCivic/dkan/pull/1824
